### PR TITLE
If only one dive selected, only one temperature in stats tab

### PR DESCRIPTION
**Pull request by avances123**
### Describe the pull request:

- [ ] Bug fix
- [x] Functional change
- [ ] New feature
- [ ] Code cleanup
- [ ] Build system change
- [ ] Documentation change
- [ ] Language translation

### Pull request long description:
In stats tab, when only one dive is selected,
on one stat, only average is shown, except temperature which
3 same temps for max, min and avg are shown.

### Changes made:
Check if combined_count is more than one to show max/min temp

### CODE REVIEW

File: desktop-widgets/tab-widgets/TabDiveStatistics.cpp [hunk 1]
```diff
@@ -69,8 +69,11 @@ void TabDiveStatistics::updateData()
 	else
 		ui->sacLimits->setAverage("");
 
-	ui->tempLimits->setMaximum(get_temperature_string(stats_selection.max_temp, true));
-	ui->tempLimits->setMinimum(get_temperature_string(stats_selection.min_temp, true));
+	if (stats_selection.combined_count > 1){
```
Comment (bstoeger) [inline desktop-widgets/tab-widgets/TabDiveStatistics.cpp:72]: What happens if the selection is changed from multiple to one dive? Then the old values remain, no?

Unless I'm missing something, I suggest adding an else branch that clears the fields in the case of a single dive.

**Review by bstoeger — CHANGES_REQUESTED** [review]
Thank you for your contribution.

Please merge the two commits into a single commit. There is no point in a whitespace-fix commit if you could simply just not introduce the whitespace problem in the first place.

File: desktop-widgets/tab-widgets/TabDiveStatistics.cpp (hunk 1 above)
Comment (avances123) [reply desktop-widgets/tab-widgets/TabDiveStatistics.cpp:72]: When you change from multiple to one dive, the new value is the only which is shown,

**Comment by avances123** [comment]
You can select the option "squash and merge"  and get this job done. Anyways I have rebased it for you.

**Review by bstoeger — APPROVED** [review]
Please add space before braces, apart from that - looks good to me.

**Comment by dirkhh** [comment]
fixed whitespace and merged this. Thanks for your contribution.

**Pull request closed without merging**